### ISSUE TITLE
Filtering doesn't work properly on Reddit

### ISSUE THREAD

**Issue by i486** [cpriest/SnapLinksPlus#282]
![snaplink](https://user-images.githubusercontent.com/625325/59957372-53199180-94ca-11e9-84d6-ece679e71053.png)

Please see the attached screenshot. It says "4 Links" but if i let go of my mouse, everything in green box will be executed.

"Disable link filtering by font weight" is still unchecked.

**Comment by cpriest** [comment]
Interesting, do you have a URL where this is happening?  Also, you are not holding down the alt key, are you?

**Comment by i486** [comment]
Yes. I am not holding down the alt key because am not using any modifier. Only right click and drag. But i just tried it again with "Alt + Right Click" setting and result is same. 

Here is the site

https://old.reddit.com/r/buildapcsales/

**Comment by i486** [comment]
Any news?

**Comment by cpriest** [comment]
I've diagnosed what's happening here:

![image](https://user-images.githubusercontent.com/159323/60433462-81822580-9bca-11e9-90e1-2343b50d5ed1.png)

Essentially, the small links (share/save/hide/report/crosspost) are of type 'Clickable' because they are javascript:void() links rather than ordinary links.  In this case, when highlighting the title and those 'Clickable' javascript links, the GreatestOfType feature is kicking in and considering the Clickable count is higher.

I'm not sure of the best way to proceed in this case.  As a workaround, if you can just highlight the main links that will open them up and ignore the javascript links for the time being.  

I'll think about how to handle this more appropriately.  It may be that I want to make <A> refs that are javascript: links be treated as just plain "Links" which would them subject them to the same font size/weight algorithm.

**Comment by i486** [comment]
Thanks for the response and explanations. 

![clicks](https://user-images.githubusercontent.com/625325/60497657-dd8e9c00-9ce7-11e9-911d-6f8181eeb720.jpg)

Do you mean like this? if yes then yeah it's a workaround I've been using for some time on Reddit  :P 

Hope you find the best way to fix this issue. Thanks!
